apps/src/netsim/NetSimVisualization.js

```diff
@@ -606,6 +576,7 @@ NetSimVisualization.prototype.getUnvisitedNeighborsOf_ = function (vizElement) {
     // Special case: The DNS node fake is a neighbor of a visited router
     if (vizElement.isRouter && this.autoDnsNode_) {
       neighbors.push(this.autoDnsNode_);
+      neighbors.push(this.autoDnsWire_);
```

Comment:
it didn't! :D
I actually spent a while trying to figure out what I had done to break that before I decided to check if it worked on the original.